assets/terraform/gce/bootstrap/centos.sh

```diff
@@ -100,16 +107,11 @@ if [ $dns_running -eq 0 ] ; then
   systemctl disable dnsmasq
 fi
 
-# According to https://stackoverflow.com/a/27667111, this is a certificate issue.
-# Although not conclusive, disabling the epel repository for the install as well
-# as the packages being installed are all from the base repository
-yum --disablerepo=epel -y update ca-certificates
-yum --disablerepo=epel -y install chrony python unzip
-
 if ! aws --version; then
-  curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
-  unzip awscli-bundle.zip
-  ./awscli-bundle/install -i /usr/local/aws -b /usr/bin/aws
+  yum -y install unzip
+  curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64-2.0.30.zip" -o "awscliv2.zip"
+  unzip awscliv2.zip
+  ./aws/install
```

Comment:
nit: iirc this installs aws with default prefixes which at least for the binary is `/usr/bin/aws-cli` instead of `/usr/bin/aws`

Reply:
The binary is available as `aws` after `./aws/install`.  Perhaps v2 changed the behavior?

Either way, the binary is showing up in the correct place.  This can been seen in my testing where the following code path:

https://github.com/gravitational/robotest/blob/459dd3903b93e12782474824f198db4974217b0c/lib/ssh/files.go#L47

is hit here: https://console.cloud.google.com/logs/query;pinnedLogId=2020-10-07T19:02:52.596557818Z%2F11quedce8ven7;query=labels.__uuid__%3D%224c928278-d561-4cec-ab4d-c9e4ce7924b7%22%0Alabels.__suite__%3D%22c139a11b-50ce-4df8-970a-41a57688bfae%22;timeRange=2020-10-07T19:01:08Z%2F2020-10-07T20:01:08Z?project=kubeadm-167321